docs/build-apps.md

```diff
@@ -713,23 +736,204 @@ client.pushCardBundleInConversationThread(carousel)
 
 ![pushCardBundleInConversationThread](./assets/images/cpa-pushCardBundleInConversationThread.png)
 
+***
+`getJWT`
+
+- Allows the CPA to get a secured JWT token. This JWT is signed with the secret token defined in the connector of the CPA.
+
+*Signature*
+
+<pre class="prettyprint">
+function getJWT(): Promise<string>
+</pre>
+
+*Example*
+<pre class="prettyprint">
+// use jwtToken on CPA for secure request
+client.getJWT().then((jwtToken: string) => { /* use JWT token */ })
+</pre>
+
+***
+
+`pushApplePayPaymentRequestInConversationThread`
+
+- Allows CPA to send Apple Pay Payment request in the conversation thread
```

Comment:
Allows the CPA to send an Apple Pay Payment request to the active conversation thread (pour être cohérent avec les lignes 621 et 678)